amulet/utils/__pipeline.py

```diff
@@ -118,12 +118,82 @@ def create_dir(path: Path | str, log: logging.Logger | None = None) -> Path:
     return resolved_path
 
 
-capacity_map = {
-    "m1": {"vgg": "VGG11", "linearnet": [128, 256, 128]},
-    "m2": {"vgg": "VGG13", "linearnet": [256, 512, 256]},
-    "m3": {"vgg": "VGG16", "linearnet": [512, 1024, 512]},
+DEFAULT_CAPACITY_MAP = {
```

Comment:
why mixed int and str?

Reply:
the str is used for MaxPool layers (or theoretically other non-standard layers)